src/app/services/auth.service.ts

```diff
@@ -1,31 +1,36 @@
-import { Auth } from '@aerogear/auth';
 import { Injectable } from '@angular/core';
-import { OpenShiftConfigService } from './config.service';
-import { KeycloakInitOptions } from 'keycloak-js';
-import { AuthContextProvider } from '@aerogear/voyager-client';
+import { ShowcaseConfigService } from './config.service';
+import { KeycloakInstance, KeycloakInitOptions } from 'keycloak-js';
+import { AuthContextProvider } from 'offix-client-boost';
 import { Platform } from '@ionic/angular';
 import { AuthStateService } from './auth-state.service';
 
+const Keycloak = require('keycloak-js');
+
 @Injectable({
     providedIn: 'root'
 })
 /**
  * Service provides Apollo Voyager client
  */
 export class AuthService {
-    public auth: Auth | undefined;
+    private keycloak: KeycloakInstance<'native'> | undefined;
     public initialized: Promise<boolean>;
     private authState: AuthStateService;
 
-    constructor(private openShift: OpenShiftConfigService, public platform: Platform, authState: AuthStateService) {
-        if (this.isEnabled()) {
-            this.auth = new Auth(this.openShift.getConfig());
+    constructor(private configService: ShowcaseConfigService, public platform: Platform, authState: AuthStateService) {
+        const config = configService.getAuthConfig();
+        if (!!config) {
+            this.keycloak = new Keycloak({ ...config });
             this.authState = authState;
             this.initialized = platform.ready().then(() => {
-                const initOptions: KeycloakInitOptions = { onLoad: 'login-required' };
-                return this.auth.init(initOptions);
+                const initOptions: KeycloakInitOptions = {
+                    onLoad: 'login-required',
+                    promiseType: 'native'
+                };
+                return this.keycloak.init(initOptions);
             }).catch((error) => {
-                console.warn(`Failed to intialize keycloak
+                console.warn(`Failed to initialize keycloak
```

Comment:
```suggestion
                console.error(error);
                console.warn(`Failed to initialize keycloak
```
We should print also the error, it is very useful to debug issues